lang/smaily-et.po

```diff
@@ -22,216 +22,281 @@ msgstr ""
 "Smaily pistikmoodulit ei saa käivitada. WooCommerce moodul on vajalik Smaily "
 "mooduli käivitamiseks. Kas WooCommerce on installitud?"
 
-#: templates/smaily-woocommerce-admin.php:25
+#: templates/smaily-woocommerce-admin.php:28
 msgid "Plugin Settings"
 msgstr "Mooduli Sätted"
 
-#: templates/smaily-woocommerce-admin.php:33
+#: templates/smaily-woocommerce-admin.php:36
 msgid "General"
 msgstr "Üldsätted"
 
-#: templates/smaily-woocommerce-admin.php:38
+#: templates/smaily-woocommerce-admin.php:41
 msgid "Customer Synchronization"
 msgstr "Kasutajate Sünkroniseerimine"
 
-#: templates/smaily-woocommerce-admin.php:43
+#: templates/smaily-woocommerce-admin.php:46
 msgid "Abandoned Cart"
 msgstr "Unustatud Ostukorv"
 
-#: templates/smaily-woocommerce-admin.php:53
+#: templates/smaily-woocommerce-admin.php:51
+msgid "Checkout Opt-in"
+msgstr "Liitumine tellimuslehel"
```

Comment:
I don't know what WooCommerce uses, but isn't "ostu vormistamine" a better term for "checkout" in Estonian?

Reply:
WooCommerce uses `Kassa leht` as translation for `Checkout` and `tellimused` for `orders`. So `Liitumine kassa lehel` would be the correct one I think.

Reply:
Ah, the magnificent world of open source translations.

Change it to `Liitumine kassa lehel` then.

Reply:
Changed to `Liitumine kassa lehel` https://github.com/sendsmaily/smaily-woocommerce-plugin/pull/45/commits/2e24bba061d5411cf28d739eab9e26a7f36d7ca7#